packages/host/app/components/operator-mode/code-submode/spec-preview.gts

```diff
@@ -446,14 +446,11 @@ export default class SpecPreview extends GlimmerComponent<Signature> {
   }
 
   isSkill(selectedDeclaration: CardOrFieldDeclaration) {
-    if (selectedDeclaration.exportName === 'SkillCard') {
-      return true;
-    }
     if (
-      selectedDeclaration.super &&
-      selectedDeclaration.super.type === 'external' &&
-      selectedDeclaration.super.name === 'SkillCard' &&
-      selectedDeclaration.super.module ===
+      selectedDeclaration.cardType.type &&
+      isResolvedCodeRef(selectedDeclaration.cardType.type.codeRef) &&
+      selectedDeclaration.cardType.type.codeRef.name === 'SkillCard' &&
+      selectedDeclaration.cardType.type.codeRef.module ===
```

Comment:
The original code was clearer that it didn't do much. 

If you want to confirm that its really an ancestor of card, you need to recurse this whole chain. 

A possible way to fix this is to introduce `specType` in card-type resource. Another way is to just recurse the chain on read and make a stopping condition

I'd recommend the 2nd way unless that is visibly inefficient